code/modules/ai/spawner_node.dm

```diff
@@ -0,0 +1,91 @@
+/obj/effect/ai_spawner_node
+	name = "AI Spawner Node"
+	icon = 'icons/effects/landmarks_static.dmi'
+	icon_state = "x6" //xxx 'X' with black borders
+	anchored = TRUE
+	invisibility = INVISIBILITY_OBSERVER
+	///Interval for how often the mobs will spawn
+	var/spawn_interval = 100
+	///Maximum amount of mobs created by this spawner that can be active
+	var/max_mobs_spawned = 10
+	///Mob we want to be spawned
+	var/mobtospawn = /mob/living/carbon/xenomorph
+	///Whether we want all the mobs in the list to be spawned or not
+	var/squad_spawner = FALSE
+
+/obj/effect/ai_spawner_node/Initialize()
+	. = ..()
+	AddComponent(AddComponent(/datum/component/spawner, mobtospawn, spawn_interval,"crawls out of", max_mobs_spawned, squad_spawner))

```

Comment:
Should this be called twice?

Reply:
Aye, that was the issue as roh told me in discord, they were also suprised that it was silently erroring